packages/web-components/src/components/ic-menu/test/basic/ic-menu.spec.tsx

```diff
@@ -20,8 +20,19 @@ const menuOptions = [
     value: "tea",
     children: [{ label: "green", value: "green" }],
   },
-  { label: "Aicano", value: "acano" },
-  { label: "Mocha", value: "mocha" },
+  {
```

Comment:
Would it be suitable to add a test for custom components in an ic-select? Something to explicitly test that the contents of a component renders, beyond passing a custom option into the regular tests? 
Might be overkill

Reply:
I think this is covered with the update to the snapshot test. If you look at the mocha/aicano part of the snapshot file, it shows the svg renders for aicano and the status tag renders for mocha

Reply:
You're right - I think I hadn't considered that since this change doesn't involve logic changes, testing the render alone is sufficient